README.md

```
@@ -333,17 +259,22 @@ docker compose up -d --build
 | **회원 목록 검색** | `GET` | `/api/v1/users` | 관리자 | QueryDSL 기반 동적 검색 및 페이징 조회 |
 | **내 프로필 수정** | `PATCH` | `/api/v1/users/me` | 인증 사용자 | 본인의 닉네임, 이름 등 프로필 정보 수정 |
 | **회원 정보 수정** | `PATCH` | `/api/v1/users/{userId}` | 관리자 | 관리자 권한으로 특정 회원 정보(역할 등) 수정 |
-| **회원 탈퇴/삭제** | `DELETE` | `/api/v1/users/{userId}` | 인증/관리자 | 회원 계정 삭제 (Soft Delete 권장) |
+| **회원 탈퇴/삭제** | `DELETE` | `/api/v1/users/{userId}` | 인증/관리자 | 회원 계정 삭제 (Soft Delete) |
 
 ### 3. 서비스 간 통신용 (Internal API)
 | 기능 | 메서드 | 경로 | 설명 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)